src/main/java/me/profelements/dynatech/DynaTech.java

```diff
@@ -68,6 +69,12 @@ public void onEnable() {
             getLogger().log(Level.WARNING, "           DynaTech will be switching to JAVA 11        ");
             getLogger().log(Level.WARNING, "                Please Update to JAVA 11                ");
         }
+
+        Bukkit.getScheduler().scheduleSyncRepeatingTask(DynaTech.getInstance(), new Runnable() {

```

Comment:
Rather than making a class with a run method and making a new Runnable to run that, either declare it all in the anonymous Runnable to make your runnable extend BukkitRunnable.

That being said, I don't think this really needs to exist for this fix which I'll explain in another comment